aten/src/ATen/native/cpu/SortingKernel.cpp

```diff
@@ -110,32 +115,73 @@ static void sort_kernel(
       int64_t dim_size
     ) {
       using scalar_t = typename std::remove_pointer<decltype(values)>::type;
+      using accscalar_t = typename AccType<scalar_t>::type;
       auto values_accessor = StridedRandomAccessor<scalar_t>(
         values, values_dim_stride);
       auto indices_accessor = StridedRandomAccessor<int64_t>(
         indices, indices_dim_stride);
-      auto composite_accessor = CompositeRandomAccessorCPU<
-        decltype(values_accessor), decltype(indices_accessor)
-      >(values_accessor, indices_accessor);
-
-      if (descending) {
-        if (stable) {
-          std::stable_sort(composite_accessor, composite_accessor + dim_size,
-            KeyValueCompDesc<scalar_t>());
+      // If the stride of the sorting dimension is not 1 or the data type of values is BFloat16,
+      // allocate a new array of accscalar_t data type and copy the values into it,
+      // and make the sorting dimension memory continuous.
```

Comment:
nit: a more common term in here is contiguous.

Reply:
I have modified it, thanks.